ofl/nanumbrushscript/METADATA.pb

```diff
@@ -1,16 +1,16 @@
-name: "Nanum Brush Script"
+name: "Nanum Brush"
```

Comment:
Can't do that! :)

Reply:
I think that it would be good to maintain consistency between "Nanum Brush Script" and "Nanum Pen". 

So either "Nanum Brush Script" and "Nanum Pen Script" or "Nanum Brush" and "Nanum Pen".

Reply:
Today on https://fonts.google.com/?subset=korean they both have "script" appended

Reply:
Oh! Well never mind me then :)

Reply:
Nanum Pen Script on GF is a bit strange.

Font name in ttf is `Nanum Pen`
![screen shot 2018-02-25 at 10 49 23](https://user-images.githubusercontent.com/7525512/36640588-da67c008-1a19-11e8-8804-b544d696e901.png)

In the metadata file we have `Nanum Pen Script`.

The font names in this pr match the font names in the fonts hosted on GF. Can we not just amend the METADATA.pb files?